crypto/armcap.c

```diff
@@ -54,6 +55,30 @@ void _armv8_pmull_probe(void);
 # ifdef __aarch64__
 void _armv8_sha512_probe(void);
 unsigned int _armv8_cpuid_probe(void);
+void _armv8_rng_probe(void);
+
+size_t OPENSSL_rndr_asm(unsigned char *buf, size_t len);
+size_t OPENSSL_rndrrs_asm(unsigned char *buf, size_t len);
+
+size_t OPENSSL_rndr_wrapper(size_t (*func)(unsigned char *, size_t), unsigned char *buf, size_t len)
```

Comment:
I don't think that any of these functions should be public, so `openssl_rndr_wrapper(...)`

Reply:
marked as static. fixed

Reply:
`_armv8_rng_probe`, `OPENSSL_rndr_asm`, `OPENSSL_rndrrs_asm(unsigned char *buf, size_t len)` were made global again since the assembly function definition was not being picked up by the compiler with strict warnings enabled. 

`static size_t OPENSSL_rndr_wrapper` has remained static since all contents definitions and references exist within this file.